ios/NetworkModule.swift

```diff
@@ -68,6 +87,35 @@ public class NetworkModule: NSObject, RCTBridgeModule {
         }
     }
 
+    internal func preprocessHttpRequestFromJS(
+        nativeId: NativeId,
+        type: String,
+        request: HttpRequest,
+        completionHandler: @escaping (
+            _ request: HttpRequest
+        ) -> Void
+    ) {
+        let requestId = "\(nativeId)@\(ObjectIdentifier(request).hashValue)"
+        let args: [Any] = [
+            requestId,
+            type,
```

Comment:
Just a note here: I see we are directly passing a String param instead of going through `RCTConvert` because that's the way it is defined in the `PreprocessHttpRequestDelegate` protocol, as opposed to what we do for the `HttpResponse` variation.

Still, it is working well in my testing 💪